pennylane/__init__.py

```diff
@@ -65,7 +65,7 @@
     shadow_expval,
 )
 from pennylane.ops import *
-from pennylane.ops import adjoint, ctrl, exp, op_sum, pow, prod, s_prod, evolve
+from pennylane.ops import adjoint, ctrl, exp, op_sum, pow, prod, s_prod, evolve, pulse
```

Comment:
Can you move `pennylane/ops/pulse` to `pennylane/pulse` as discussed?

Reply:
Also move the tests please

Reply:
Yes! Was just waiting on that PR to be merged to be sure its clear where everything goes, will reorganize now :)